pkg/api/handlers/compat/auth.go

```diff
@@ -50,9 +50,19 @@ func Auth(w http.ResponseWriter, r *http.Request) {
 			Status:        "Login Succeeded",
 		})
 	} else {
-		utils.WriteResponse(w, http.StatusBadRequest, entities.AuthReport{
-			IdentityToken: "",
-			Status:        "login attempt to " + authConfig.ServerAddress + " failed with status: " + err.Error(),
+		var msg string
+
+		var unauthErr DockerClient.ErrUnauthorizedForCredentials
+		if errors.As(err, &unauthErr) {
```

Comment:
I think this should use `errors.Is`

Reply:
@Luap99
AFAIK `Is()` is used when checking against instance (sentinel error) and `As()` when checking against type. I ma checking against type.

Reply:
ErrUnauthorizedForCredentials is `type` not `var`

Reply:
ok, sorry for the noise.